platforms/BUCK

```diff
@@ -0,0 +1,14 @@
+[
+    platform(
```

Comment:
oh interesting you're also doing platforms... what does the `config_setting` in that other file accomplish then?

Reply:
i think you need both? if i remove one or the other it fails, and i dont remember but a guide or the docs had you put both

Reply:
discussed offline: both are required, we're not entirely sure why, who cares